scipy/optimize/_cobyqa_py.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from ._optimize import _check_unknown_options
+
+__all__ = []
```

Comment:
I think you can probably just remove this line to make mypy happy
```suggestion
```

Reply:
Thank you very much for your suggestion, I was not sure how to deal with mypy. I just made a new commit to also remove the empty line above and avoid linting issues.

Cheers,
Tom.